pom.xml

```diff
@@ -57,6 +57,7 @@
         <dep.assertj-core.version>3.8.0</dep.assertj-core.version>
         <dep.logback.version>1.2.3</dep.logback.version>
         <dep.jmxutils.version>1.20</dep.jmxutils.version>
+        <dep.gcs.version>1.9.10</dep.gcs.version>
```

Comment:
Looks like the current version is `1.9.13`. Should we try that one?

Reply:
`1.9.10` is a stable version that has been tested and verified in our production environment. I am OK using the latest version, but I am not sure if it is stable or not.

Reply:
Makes sense. Let's continue using the tested version.